recipes-ni/base-files-nilrt/base-files-nilrt_1.0.bb

```diff
@@ -69,4 +72,7 @@ do_install () {
 
 	# add sysctl.conf file to adjust system configuration parameters
 	install -m 0644 ${WORKDIR}/sysctl.conf ${D}${sysconfdir}/
+
+	# add machine-info and allow System Web Server to modify it
+	install -m 0664 -g ni ${WORKDIR}/machine-info ${D}${sysconfdir}/
```

Comment:
I think `LVRT_GROUP = "ni"` should be available as a variable via the [distro conf](https://github.com/ni/meta-nilrt/blob/nilrt/master/dunfell/conf/distro/nilrt.inc#L27). Is it available for you to use in the recipe here, instead of hard-coding the `ni` value?

Reply:
Good call; I forgot about that var.